frontend/packages/noobaa-storage-plugin/src/components/bucket-class/create-bc.tsx

```diff
@@ -106,20 +121,50 @@ const CreateBucketClass: React.FC<CreateBCProps> = ({ match }) => {
 
   return (
     <>
-      <Wizard
-        isCompactNav
-        isInPage
-        isOpen
-        title="Create new Bucket Class"
-        description="NooBaaBucketClass is a CRD representing a class for buckets that defines policies for data placement and more"
-        steps={steps}
-        onSave={finalStep}
-        onClose={() => history.goBack()}
-      />
+      <div className="nb-create-bc-header">
+        <div className="nb-create-bc-header__breadcrumbs">
+          {clusterServiceVersion !== null && (
+            <BreadCrumbs
+              breadcrumbs={[
+                {
+                  name: _.get(
+                    clusterServiceVersion,
+                    'spec.displayName',
+                    'Openshift Container Storage Operator',
+                  ),
+                  path: match.url.replace('/~new', ''),
+                },
+                { name: `Create ${NooBaaBucketClassModel.label}`, path: match.url },
+              ]}
+            />
+          )}
+        </div>
+        <div className="nb-create-bc-header-title">
+          <Title size="2xl" headingLevel="h1" className="nb-create-bc-header-title__main">
+            Create new BucketClass
+          </Title>
+          <p className="nb-create-bc-header-title__info">
+            NooBaaBucketClass is a CRD representing a class for buckets that defines policies for
+            data placement and more
+          </p>
+        </div>
+      </div>
+      <div className="nb-create-bc-wizard">
+        <Wizard
+          isCompactNav
+          isInPage
+          isOpen
+          title="Create new Bucket Class"
+          description="NooBaaBucketClass is a CRD representing a class for buckets that defines policies for data placement and more"
```

Comment:
1. Is `NooBaaBucketClass` a single word?
2. data placement and more? what `more`? doesn't sound right to me.
```suggestion
          description="NooBaaBucketClass is a CRD representing a class for buckets that defines policies for data placement and more".
```

Reply:
![Screenshot from 2019-11-06 12-31-11](https://user-images.githubusercontent.com/54092533/68275342-5b492680-0091-11ea-9765-dd948e003ecd.png)
From the design doc

Reply:
@shirimordechay ^^

Reply:
@bipuladh We should change any NooBaaBucketClass reference to BucketClass so it will match the CRD. It should be presented as one word (BucketClass). 

Regarding the description text, we can change it to: "BucketClass is a CRD representing a class for buckets that defines tiering policies and data placement"